Box_intersection_d/doc/Box_intersection_d/PackageDescription.txt

```diff
@@ -2,6 +2,9 @@
 /// \defgroup PkgBoxIntersectionDConcepts Concepts
 /// \ingroup PkgBoxIntersectionDRef
 
+/// \defgroup PkgBoxIntersectionDEnums Rnumerations
```

Comment:
`Rnumerations` -> `Enumerations`

Reply:
stupid me.
I am also wondering if we do not have to add a `\cgalCRPSection{Enumerations}`.   Do we have a canonical order for those? In Arrangements the `enums` come before the concepts.

Reply:
When you mean with canonical something like `cgalEnumerations` I don't think so. A grep over `BaseDoxyfile.in` for `cgalCRPSection` just gave `cgalCRPSection{1}=<h2>\1</h2>`.
There are quite a few usages of `cgalCRPSection` though and when just looking for enumeration types I found the usages:
- Enum
- Enums
- Enumerations
- Constants and Enumerations

This cannot be correct (I think):
```
Arrangement_on_surface_2/doc/Arrangement_on_surface_2/PackageDescription.txt:\cgalCRPSection{Enumerations}
Circular_kernel_3/doc/Circular_kernel_3/PackageDescription.txt:\cgalCRPSection{Constants and Enumerations}
Cone_spanners_2/doc/Cone_spanners_2/PackageDescription.txt:\cgalCRPSection{Enumerations}
Kernel_23/doc/Kernel_23/PackageDescription.txt:\cgalCRPSection{Constants and Enumerations}
Mesh_2/doc/Mesh_2/PackageDescription.txt:\cgalCRPSection{Enumerations}
Mesh_3/doc/Mesh_3/PackageDescription.txt:\cgalCRPSection{Enumerations}
Periodic_2_triangulation_2/doc/Periodic_2_triangulation_2/PackageDescription.txt:\cgalCRPSection{Enums}
Periodic_3_triangulation_3/doc/Periodic_3_triangulation_3/PackageDescription.txt:\cgalCRPSection{Enums}
Ridges_3/doc/Ridges_3/PackageDescription.txt:\cgalCRPSection{Enums}
Stream_support/doc/Stream_support/PackageDescription.txt:\cgalCRPSection{Enum}
Surface_mesh_shortest_path/doc/Surface_mesh_shortest_path/PackageDescription.txt:\cgalCRPSection{Enums}
Triangulation/doc/Triangulation/PackageDescription.txt:\cgalCRPSection{Enums}
Triangulation_2/doc/Triangulation_2/PackageDescription.txt:\cgalCRPSection{Enum}
Triangulation_3/doc/Triangulation_3/PackageDescription.txt:\cgalCRPSection{Enums}
Triangulation_on_sphere_2/doc/Triangulation_on_sphere_2/PackageDescription.txt:\cgalCRPSection{Enum}
```

In all there I quickly found 391 usages of `cgalCRPSection` with 168 unique values, I used (a.o.):
```
grep cgalCRPSection -r [A-Z]* | grep -v "Documentation"  | sed -e "s/[^:]*://" | sort | uniq
```
and see that there are more catagories with "double" names (see e.g.g Traits...)

Reply:
I was more wondering about the order of the sections
![image](https://github.com/CGAL/cgal/assets/3263539/4e942f97-876e-47d0-8f8a-da8348ad0056)

Reply:
The order is something doxygen cannot enforce and has to be done by means of "coding standards" and can be checked by part the "grep" I gave (i.e. the part: `grep cgalCRPSection -r [A-Z]* | grep -v "Documentation"`) and checking whether or not the order is OK (script).